assets/components/emailFormField/emailFormField.jsx

```diff
@@ -4,33 +4,58 @@
 
 import React from 'react';
 import TextInput from 'components/textInput/textInput';
+import ErrorMessage from 'components/errorMessage/errorMessage';
+import { validateEmailAddress } from '../../helpers/utilities';
 
 // ----- Types ----- //
 
 type PropTypes = {
   emailUpdate: (email: string) => void,
   email: string,
   isSignedIn: boolean,
+  setEmailHasBeenBlurred: () => void,
+  emailHasBeenBlurred: boolean,
```

Comment:
This should be causing validation errors in the one-off container for `EmailFormField` shouldn't it? I wonder why flow isn't picking it up...

Reply:
@Ap0c I have now implemented the validation for one off too, so this isn't an issue any more